xml/art_sle_ha_pmremote.xml

```diff
@@ -0,0 +1,474 @@
+      </step>
+      <step>
+       <para>Restart &pace; using the <command>systemctl restart pacemaker</command> command.</para>
+      </step>
+      <step>
+       <para>Add the following line in the <filename>/etc/csync2/csync2.cfg</filename> file:</para>
+       <screen>include /etc/pacemaker/authkey</screen>
+      </step>
+      <!-- Run csync2? -->
+     </substeps>
+    </step>
+   </procedure>
+  </sect2>
+
+  <sect2 xml:id="sec.ha.pmremote.config.kvm.guest">
+   <title>Configuring the KVM Guests</title>
+   <para>
+    The following procedure creates a KVM guest node <systemitem
+      class="domainname">&wsII;</systemitem> on your physical host
+    <systemitem class="domainname">&node5;</systemitem>:
+   </para>
+   <procedure>
+    <step>
+     <para>Create a KVM guest on <systemitem class="domainname"
+      >&node5;</systemitem>. For details refer to <citetitle>&deploy;</citetitle>.
+      Make sure to configure the guest with a host name and a static IP address. </para>
+    </step>
+    <step>
+     <para>Add the KVM guest host name <systemitem
+      class="domainname">&wsII;</systemitem> and its IP address to the hosts
+     machine at <filename>/etc/hosts</filename>.</para>
+    </step>
+    <step>
+     <para>Configure Firewall settings on your guest node and open TCP
+      port 3121.</para>
+    </step>
+    <step>
+     <para>Verify connectivity between guests and host by using
+      <command>ping</command> and <command>ssh</command>.</para>
+    </step>
+    <step>
+     <para>Install the <package>pacemaker-remote</package> package and
+     enable it:</para>
+     <screen>&prompt.root;<command>zypper</command> in pacemaker-remote
+&prompt.root;<command>systemctl</command> enable pacemaker_remote</screen>
+    </step>
+    <step>
+     <para>Create the <filename>/etc/pacemaker</filename> directory and change
+      the group:</para>
+     <screen>&prompt.root;<command>mkdir</command> -p --mode=0750 /etc/pacemaker
+&prompt.root;<command>chgrp</command> haclient /etc/pacemaker</screen>
+    </step>
+    <step>
+     <para>Run <command>csync2</command> to copy the authentication key from
+      your physical host to your guest node:</para>
+     <screen>&prompt.root;<command>csync</command> -xv</screen>
+    </step>
+    <step>
+     <para>Verify host connection to guest. This can be done by
+      making a SSH connection from the host. This connection
+      will fail, but <emphasis>how</emphasis> it fails gives you hints
+      about its connectivity.
+     </para>
+     <para>Run <command>ssh</command> on one of the cluster nodes:</para>
+     <screen>&prompt.root;<command>ssh</command> -p 3121 &wsII;</screen>
+     <para>An error connection message should be as follows:</para>
+     <screen>ssh_exhange_identification: read: Connection reset by peer.</screen>
+     <para>If you see either of the following messages, the connection does
+      not work:</para>
```

Comment:
you said above: 'but <emphasis>how</emphasis> it fails gives you hints about its connectivity'.
But what do those messages tell the admin now (other than that the connection fails)? The 'how' part is missing here.